interface/usergroup/user_admin.php

```diff
@@ -559,7 +565,56 @@ function authorized_clicked() {
   </select></td>
   <td><span class=text><?php echo xlt('Additional Info'); ?>:</span></td>
   <td><textarea style="width:150px;" name="comments" wrap=auto rows=4 cols=25><?php echo $iter["info"];?></textarea></td>
+  </tr>
+  <tr>
+  <td><span class="text"><?php echo xlt('Full-screen role:'); ?></span></td>
```

Comment:
If the punctuation is needed put it outside of the xlt so it will translate with no issue. ('Full screen role'); ?> :

Reply:
Okay will fix that